libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -113,10 +113,15 @@ export const OcmClusterDetailsFormFields = ({
   }, []);
 
   React.useEffect(() => {
-    if (!oracleDropdownItemState?.isSupported) {
-      setFieldValue('platform', 'none');
+    if (clusterPlatform !== undefined) {
+      const platform = clusterPlatform === 'baremetal' ? 'none' : clusterPlatform;
```

Comment:
Given the new discussion, I don't think we should be swapping `baremetal` (which would be the new default) for `none`.  This seems to be the reverse of the code in the submit action.

This should be handled from the explicit actions we have discussed.